### DISCUSSION TITLE
3.5” 8 Bit parallel interface (ILI9486) display with ESP32. Touch functionality?

### DISCUSSION THREAD

**Discussion by imanurebak** [Bodmer/TFT_eSPI#3356] [Q&A - Touch]
Hello, thanks for the library. I would like to know if you have tested the touch functionality of the 3.5” 8 Bit parallel display with the ESP32. The display is displayed, but when compiling code that includes touch functionality code, it no longer compiles. Compilation errors like: src/main.cpp:121:22: error: 'class TFT_eSPI' has no member named 'getTouch'.  I already tried different User_Setup.h configurations.

The display is this: https://es.aliexpress.com/item/32585602722.html?gatewayAdapt=glo2esp
It has ili9486 driver.

I would appreciate any help.

**Comment by elgerg** [comment]
Hi,

You should have searched the issue list for this error:
https://github.com/Bodmer/TFT_eSPI/issues?q=has+no+member+named+getTouch

Basically the library only supports one touch controller and thats when it's on the same SPI bus as the display. As yours is parallel it isnt supported. You will need to use another library specifically for the touch component.

**Comment by imanurebak** [comment]
Ok, you have been very clear in your answer. I will try to make the touch work with some other library. Thank you very much.

**Reply by RDJunc** [reply]
Have you solved this problem and if so, what library did you use to implement the touch?

**Reply by imanurebak** [reply]
Yes. https://www.youtube.com/watch?v=EiC2gSOasJw
